src/PwdResetEmailSentController.js

```diff
@@ -14,7 +14,7 @@ define([
   'okta',
   'util/Enums',
   'util/FormController',
-  'util/FormType'
+  'util/FormType',
```

Comment:
Was the comma `,` added for a lint error?

Reply:
I think it doesn't trigger any lint error.